<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ashok" name="com.mst.selenium.main.test.TestEngine" tests="8" failures="0" timestamp="8 Mar 2017 22:32:43 GMT" time="501.438" errors="2">
  <testcase name="leadCreation" time="71.698" classname="com.mst.selenium.main.test.TestEngine"/>
  <testcase name="leadTabVerification" time="54.891" classname="com.mst.selenium.main.test.TestEngine"/>
  <testcase name="contactCreation" time="62.784" classname="com.mst.selenium.main.test.TestEngine"/>
  <testcase name="leadCreation" time="59.682" classname="com.mst.selenium.main.test.TestEngine"/>
  <testcase name="contactCreation" time="80.307" classname="com.mst.selenium.main.test.TestEngine"/>
  <testcase name="accountCreation" time="66.368" classname="com.mst.selenium.main.test.TestEngine">
    <error type="com.mst.selenium.exception.CustomException" message="The value Edge Communicationsis not matched with the given value Edge Communication">
      <![CDATA[com.mst.selenium.exception.CustomException: The value Edge Communicationsis not matched with the given value Edge Communication
	at com.mst.selenium.main.test.TestEngine.highlightElementBasedOnResult(TestEngine.java:142)
	at com.mst.selenium.main.test.TestEngine.accountCreation(TestEngine.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- accountCreation -->
  <testcase name="leadTabVerification" time="46.936" classname="com.mst.selenium.main.test.TestEngine"/>
  <testcase name="accountCreation" time="58.772" classname="com.mst.selenium.main.test.TestEngine">
    <error type="com.mst.selenium.exception.CustomException" message="The value Edge Communicationsis not matched with the given value Edge Communication">
      <![CDATA[com.mst.selenium.exception.CustomException: The value Edge Communicationsis not matched with the given value Edge Communication
	at com.mst.selenium.main.test.TestEngine.highlightElementBasedOnResult(TestEngine.java:142)
	at com.mst.selenium.main.test.TestEngine.accountCreation(TestEngine.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- accountCreation -->
</testsuite> <!-- com.mst.selenium.main.test.TestEngine -->
